src/OpenTelemetry/CHANGELOG.md

```diff
@@ -42,6 +42,10 @@
   [IMetricsListener](https://learn.microsoft.com/dotNet/api/microsoft.extensions.diagnostics.metrics.imetricslistener).
   ([#5265](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5265))
 
+* **Experimental (pre-release builds only):** `Exemplar` and `ExemplarReservoir`
+  APIs have been updated to match the OpenTelemetry Specification.
```

Comment:
Is there a practice to provide OpenTelemetry Specification link in the changelog?

Reply:
I added some links in there for you: https://github.com/open-telemetry/opentelemetry-dotnet/pull/5386/commits/5f04c40153c164d9c5b83464a7afe2a8a50e910a